database.php

```diff
@@ -0,0 +1,11 @@
+<?php
```

Comment:
@LIGHT1210 As we talked this needs to be changed to a single events.sqlite file instead of using MySQL server.